client/packages/system/src/ContactTrace/DetailView/Footer.tsx

```diff
@@ -69,14 +72,16 @@ export const Footer: FC<FooterProps> = ({
               }}
               label={t('button.cancel')}
             />
-            <ButtonWithIcon
-              variant="outlined"
+            <LoadingButton
               color="secondary"
-              Icon={<CheckIcon />}
+              variant="outlined"
```

Comment:
would you mind adding

```
              loadingStyle={{ iconColor: 'secondary.main' }}
```

currently the loader looks odd with the primary colour: 

<img width="304" alt="Screenshot 2024-01-08 at 10 54 12 AM" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/74b2caed-e085-4372-aa30-b6c9c4dc8277">